haiku-libs/sagebrush/sagebrush-1.0.0.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="An interface library for image editor"
+DESCRIPTION="SageBrush is an interface library for image editor. It provides \
+a kit to build an image editor easily without starting from scratch."
+HOMEPAGE="https://github.com/HaikuArchives/SageBrush"
+COPYRIGHT="2012 Vladislav Burundukov"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="d9b3945cb78c7070aa8b48d2e0bc6a2007fa3601"
+SOURCE_URI="https://github.com/HaikuArchives/SageBrush/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d152e0d572a8eb0b8fc39a531cc1466827a9b96526bc5669dd181ef77ed32940"
+SOURCE_FILENAME="SageBrush-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="SageBrush-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	sagebrush$secondaryArchSuffix = $portVersion
```

Comment:
Currently you have no secondary arch defined in your recipe, and this won't build with gcc5 so you can remove all the $secondaryArchSuffix safely.

Reply:
Or you can introduce it like:
SECONDARY_ARCH="!x86"
in the line 14.